mule-extensions-api-persistence/src/main/java/org/mule/runtime/extension/internal/persistence/ErrorModelSerializerDelegate.java

```diff
@@ -103,7 +106,9 @@ private void writeError(JsonWriter out, ErrorModel errorModel) throws IOExceptio
       if (error.has(PARENT)) {
         parentError = error.get(PARENT).getAsString();
       }
-      buildingErrors.put(anError, new Pair<>(parentError, newError(buildFromStringRepresentation(anError))));
+      ErrorModelBuilder errorModelBuilder = newError(buildFromStringRepresentation(anError))
+          .handleable(error.has(HANDLEABLE) ? error.get(HANDLEABLE).getAsBoolean() : DEFAULT_HANDLEABLE);
```

Comment:
remove the "DEFAULT", if `HANDLEABLE` is not present just don't set it and let the builder decide the default.